src/go/k8s/cmd/configurator/main.go

```diff
@@ -40,20 +40,24 @@ const (
 	externalConnectivityEnvVar          = "EXTERNAL_CONNECTIVITY"
 	externalConnectivitySubDomainEnvVar = "EXTERNAL_CONNECTIVITY_SUBDOMAIN"
 	hostPortEnvVar                      = "HOST_PORT"
+	proxyExternalConnectivityEnvVar     = "PROXY_EXTERNAL_CONNECTIVITY"
+	proxyHostPortEnvVar                 = "PROXY_HOST_PORT"
 )
 
 type brokerID int
 
 type configuratorConfig struct {
-	hostName             string
-	svcFQDN              string
-	configSourceDir      string
-	configDestination    string
-	nodeName             string
-	subdomain            string
-	externalConnectivity bool
-	redpandaRPCPort      int
-	hostPort             int
+	hostName                  string
+	svcFQDN                   string
+	configSourceDir           string
+	configDestination         string
+	nodeName                  string
+	subdomain                 string
+	externalConnectivity      bool
```

Comment:
this is not for this PR but we should change this, there's no global external connectivity anymore so the name of this at least internally should change

Reply:
Agree. Also "listener" across the codebase could become "kafkaAPIListerner" like other APIs.